apps/api/src/exception-filter.ts

```diff
@@ -110,6 +119,24 @@ export class ErrorDto {
   errorId?: string;
 
   path: string;
-
   message: string | object;
 }
+
+function handleZod(exception: ZodError<any>) {
+  const status = HttpStatus.BAD_REQUEST; // Set appropriate status for ZodError
+  const message = {
+    errors: exception.errors.map((err) => ({
+      message: err.message,
+      path: err.path,
+    })),
+  };
+
+  return { status, message };
+}
+
+function handleCommandValidation(exception: CommandValidationException) {
+  const { mappedErrors } = exception;
+  const { message } = exception;
+
+  return { message: { message, cause: mappedErrors }, status: 400 };
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Looks like we can use the enum here too.‏
```suggestion
  return { message: { message, cause: mappedErrors }, status: HttpStatus.BAD_REQUEST };
```